src/firefly/js/core/background/BgMaskPanel.jsx

```diff
@@ -28,13 +28,20 @@ export class BgMaskPanel extends SimpleComponent {
             bgStatus && dispatchJobAdd(bgStatus);
         };
 
+        const button = bgStatus && bgStatus.isLocal && bgStatus.isLocal?
+            <button type='button' style={maskButton} className='button std' onClick={sendToBg}>Please click to send to background</button>
+            :
+            <button type='button' style={maskButton} className='button std'> Saving to workspace, please wait...</button>;
+
+
         if (inProgress) {
             return (
                 <div style={maskStyle}>
                     <div className='loading-mask'/>
                     {bgStatus &&
                     <div style={{display: 'flex', alignItems: 'center'}}>
-                        <button type='button' style={maskButton} className='button std' onClick={sendToBg}>Send to background</button>
+ {/*                       <button type='button' style={maskButton} className='button std' onClick={sendToBg}>Please click to send to background</button>
+*/}                  {button}
```

Comment:
So now `Send to background` is removed for everything?  Is it still available for catalog search?